recipes/drwav/all/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.4)
+project(dr_wav C)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+option(NO_CONVERSION_API "Build with define DR_WAV_NO_CONVERSION_API" OFF)
+option(NO_STDIO "Build with define DR_WAV_NO_STDIO" OFF)
+
+add_library(${CMAKE_PROJECT_NAME} dr_wav.c)
+
+target_include_directories(${CMAKE_PROJECT_NAME} PRIVATE source_subfolder)
+
+set_target_properties(${CMAKE_PROJECT_NAME} PROPERTIES 
+    PUBLIC_HEADER source_subfolder/dr_wav.h
```

Comment:
```suggestion
    DEFINE_SYMBOL DRWAV_DLL
    PUBLIC_HEADER source_subfolder/dr_wav.h
```

Reply:
@madebr Are you sure `DEFINE_SYMBOL` is equivalent to 
```cmake
if(BUILD_SHARED_LIBS)
    target_compile_definitions(${CMAKE_PROJECT_NAME} PUBLIC -DDRWAV_DLL)
endif()
```
?

From the description at https://cmake.org/cmake/help/latest/prop_tgt/DEFINE_SYMBOL.html, I understand this would only be set when compiling the sources (for dllexport) to a shared library but not when linking (for dllimport). But in this case, the define should be always set, as there is a separate define (`DRWAV_IMPLEMENTATION`) for discerning exporting from importing.

Reply:
You're right.
`DEFINE_SYMBOL` is inly added when the shared library, not when consuming.
If it's needed in both cases, you need to do `target_compile_definitions`.

Reply:
Can be ignored after https://github.com/conan-io/conan-center-index/pull/7927#discussion_r844271697